### DISCUSSION TITLE
Tasmota control of Zigbee devices

### DISCUSSION THREAD

**Discussion by wayner9** [arendst/Tasmota#19974] [General]
How does Tasmota work with Zigbee devices, like some of the Sonoff Zigbee sensors?  Do you just flash Tasmota on the Zigbee bridge and not flash the individual Zigbee devices like sensors/switches?  Or do you have to flash those individual devices as well?

**Comment by sfromis** [comment]
https://tasmota.github.io/docs/Zigbee/

Basically, you do not do anything to the end devices, but have a coordinator (that "bridge") able to run Tasmota.

**Reply by wayner9** [reply]
Thanks, I had skimmed through that web page but it wasn't clear to me if you needed to do anything to the end devices.  Does the bridge have to be the same brand as the end devices, or can you use a "generic" bridge.

**Reply by sfromis** [reply]
No need for same brand; to run Tasmota, you just need a coordinator (this is the Zigbee term) compatible with Tasmota. While most end devices work without trouble, others may be "less straightforward", especially if they break standards by doing Tuya stuff instead.

**Reply by wayner9** [reply]
Any suggestions on what Zigbee coordinator that I should get to run Tasmota.  I mentioned the Sonoff one, but are there others that would be suitable?

**Reply by sfromis** [reply]
This one has Ethernet and CC2652P, both are a plus to have, but not "needed".
https://templates.blakadder.com/athom_ZG01.html
